.github/workflows/test-js.yml

```diff
@@ -0,0 +1,21 @@
+name: JS tests
+on:
+  # Run this on push to master
+  push:
+    branches:
+      - master
+  # Run this on push to PRs, regardless of branch
+  pull_request:
+  # [Monorepo] Only run this if there are changes to web
+  # paths:
+  #   - "web/**"
```

Comment:
This is commented out, the `pull_request:` object is empty. Is this valid YAML?

Reply:
It's valid: ["You must append a colon (:) to all events, including events without configuration."](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#on)

Commented out section is explained in the OP

Reply:
For reference, when that section is uncommented, the indentation of that snippet would look like this:
```
pull_request:
paths:
  - "web/**"
```